doc/claude/self.md

```
@@ -1,414 +1,414 @@
+const { name, age, email } = user;
+const [first, second, ...rest] = items;
+
+// ✅ 説明的命名（短縮禁止）
+const isUserAuthenticated = true;  // ✅ 分かりやすい
+const isAuth = true;               // ❌ 短縮形禁止
+```
+
+## 📁 コロケーション戦略（Locality of Behavior）
+
+### ファイル配置の哲学
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)